index.html

```diff
@@ -1,12 +1,14 @@
 ---
 layout: question
-title: "Is it on contract"
+title: "Explore the Pittsburgh Procurement Process!"
 display_title: "Explore"
+permalink: "/"
 buttons:
   -
-    text: "Yes"
-    url: use-it
+    text: "Get Started!"
+    url: is-it-on-contract
+pre_button_content:
   -
-    text: "No"
-    url: what-are-you-sourcing
+    title: "What is this?"
+    content: "This tool is intended to help you navigate which process you should use to purchase an item for the City of Pittsburgh. Just answer each question in sequence until you end up at a recommendation page. Occasionally, it will also link you to outside resources to help you make a decision, answer a question, or learn more about a specific process."
```

Comment:
"Wondering what steps to take to purchase something for the City of PIttsburgh? Answer a couple quick questions and find out what purchasing process to use."